oic.r.csr.raml

```diff
@@ -0,0 +1,35 @@
+#%RAML 0.8
+title: OICSecurityCsrResource
+version: v1.0-20150819
+schemas:
+  - Csr: !include schemas/oic.r.csr.json
+
+traits:
+  - interface:
+      queryParameters:
+        if:
+            enum: ["oic.if.baseline"]
+
+/oic/sec/csr:
+  description: |
+    This resource specifies a Certificate Signing Request.
+
+  displayName: Certificate Signing Request
+  is: [ interface ]
+
+  get:
+    description: |
+      Retrieves CSR data.
+    responses:
+      200:
+        body:
+          application/json:
+            schema: Csr
+            example: |
+              {
+              "rt": ["oic.r.csr"],
+              "encoding" : "oic.sec.encoding.pem"
+              "csr": "PEMENCODEDCSR"
+              }
+      503:
```

Comment:
BTW, there's some interconnection here with the fixed URI discussion that occurred during the Amsterdam face-to-face.

If the scr resource was required to be discovered via the res resource, then non-presence of scr in res could indicate non-support. Given a fixed-path URI for scr (which I think is SWG's position for OCF 1.0) - and the CR indicates that cert support is a necessary but not sufficient condition for supporting scr - then returning something like 404 seems like the best option for indicating non-support for scr, don't you think?